libs/sdk-core/src/models.rs

```diff
@@ -816,6 +816,36 @@ pub struct RefundResponse {
     pub refund_tx_id: String,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct FetchInvoiceRequest {
+    pub offer: String,
+    pub amount_msat: Option<u64>,
+    pub timeout: Option<f64>,
+    pub payer_note: Option<String>,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct FetchInvoiceResponse {
+    pub bolt12: String,
+    pub new_amount_msat: Option<u64>,
+}
```

Comment:
These are not publicly exposed right but only needed internally in the sdk right? If so let's change their access modifier accordingly.